### DISCUSSION TITLE
关于定时运行的频率

### DISCUSSION THREAD

**Discussion by mapxn** [luolongfei/freenom#83] [Ideas]
Freenom免费续期时间为，域名到期前14天内。所以把频率设置为一周一次也可以满足需求，这样降低了大家同时运行的概率。

比如`33 12 * * 1` --> 每周一12:33运行一次

有个很方便的cron表达式在线工具，https://crontab.guru/

**Comment by luolongfei** [comment]
这个频率是可以自定义的，RUN_AT 可以直接指定为 cron 表达式。

**Reply by mapxn** [reply]
对的，运行时间可以自定义。毕竟大部分人可能都是使用默认，建议默认设置为 **一周内的随机时间点** ，减少撞车概率嘛。
（小建议，采不采纳无所谓）

**Reply by luolongfei** [reply]
默认的话是自动指定一个随机时间点。每天执行实际上是有一个考量，就是当 freenom 的页面结构或者续期逻辑改变，当天执行的时候就会送信错误信息，可以第一时间发现问题。每周执行的话后续版本也可以考虑。
